scss/main.scss

```diff
@@ -31,38 +31,17 @@ p {
 }
 
 .btn-link {
-  cursor: pointer;
-  padding: 0;
+  &.inline-link {
```

Comment:
Doesn't this just make it look like a regular anchor element? Can we just use `<a href="/">` with no classes?